package utils.pojo;

import java.time.LocalDateTime;

public class User {
    private String email;
    private String password;
    private String name;

    public User(String email, String password, String name) {
        this.email = email;
        this.password = password;
        this.name = name;
    }

    public static User random() {
        var rnd = LocalDateTime.now().getNano();
        return new User("test" + rnd + "@gmail.ru", "testtest", "test" + rnd);
    }

    public String getEmail() {
        return email;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }
}
